types/src/database_sql.rs

```diff
@@ -666,17 +680,39 @@ impl Database {
         Ok(media_slice_id)
     }
 
-    /// Update if a piece of audio will be shown to readers
+    /// As above in `attach_audio_to_word`:
+    /// Creates media slice if doesn't yet exist for provided audio recording.
+    /// Adds join table entry attaching media slice to document.
+    /// Returns `id` of upserted media slice.
+    pub async fn attach_audio_to_document(
+        &self,
+        upload: &AttachAudioToDocumentInput,
+        contributor_id: &Uuid,
+    ) -> Result<Uuid> {
+        let media_slice_id = query_file_scalar!(
+            "queries/attach_audio_to_document.sql",
+            contributor_id,
+            upload.contributor_audio_url as _,
+            0,
+            0,
```

Comment:
Is there a particular reason you set the time bounds to 0 here as opposed to in the SQL query?

Reply:
I wanted to start by mirroring the way we were already doing it in attach_audio_to_word above on lines 665-681. Do you think we should move some things around in this case?

Reply:
No I think this is fine! I just wanted to know where your thinking was.